doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,15 +394,36 @@ ZTA principles applied to Cloud infrastructure components are the following:
 
-Open Source Software Security
------------------------------
+Software Supply Chain Security
+------------------------------
+
+Software supply chain attacks are increasing worldwilde and can cause serious damages. Many entreprises and 
+organisations are experiencing these threats. Software suppliers can be targeted and by rebound effect their 
+customers, as seen with the backdoors affecting SolarWinds products and detailed in
+`Defending against SolarWinds attacks 
+<https://www.techtarget.com/searchsecurity/news/252494495/Defending-against-SolarWinds-attacks-What-can-be-done>`_.
+Open-source code weaknesses can also be utilised by attackers, the Log4J vulnerability, impacting lots of 
+applications, is a recent example in this field. The cyber security of software supply chain are often not 
+taken into account. Gouvernments are alerting and requesting actions to face these risks. The British 
+government is hardening the law and standards of cyber security for the supply chain. The US government 
+requested actions to enhance the software supply chain security. The security of software supply chain 
+is a challenge pointed out by the European Network and Information Security Agency, ENISA, 
+in the report `NFV Security in 5G - Challenges and Best Practices 
+<https://www.enisa.europa.eu/publications/nfv-security-in-5g-challenges-and-best-practices>`_.
+ 
+
+Software security
+~~~~~~~~~~~~~~~~~
 
-Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments. Open source code is present in Cloud Infrastructure software from host Operating System to virtualisation layer components, the most obvious being represented by Linux, KVM, QEMU, OpenStack, and Kubernetes. Workloads components can also be composed of open source code. The proportion of open source code to an application source code can vary. It can be partial or total, visible or not. Open source code can be upstream code coming directly from open source public repositories or code within a commercial application or network function. To ensure the security of the whole system, all software and hardware components must reach the same level of security by following best security practices including secure lifecycle management. The SAFECode paper “Managing Security Risks Inherent in the Use of Third-party Components” provides a detailed risk management approach.
+Software supply chain safety is crucial and can be a complex task in virtualised and containerised environments
```

Comment:
Should be security, not safety.

Reply:
@ulikleber I think that both work

Reply:
@karinesevilla , @ulikleber: this is a very subtle linguistic difference, and none of us was born in an English speaking country (I guess...).  I would refer us to an interesting discussion in  http://www.differencebetween.net/language/words-language/difference-between-safety-and-security/.  My  feeling from reading the comparison there, that "security" would be more appropriate in our case.

Reply:
May I suggest a compromise and we use "security and safety"?

Reply:
I think using safety when talking about software vulnerability is really strange. 
Safety in the context of computers is for instance used, when you program a robot that when moving around could hurt people. So the software reaction times and reliability are critical.

Reply:
Thanks all for sharing your opinions. There is a consensus in favor of "security". I modified the text.

Reply:
Thanks for sharing your opinions. There is a consensus to use the word security, I made the modification